clang/test/CodeGenSYCL/loop_unroll_host.cpp

```diff
@@ -2,6 +2,14 @@
 // CHECK: br label %{{.*}}, !llvm.loop ![[COUNT:[0-9]+]]
 // CHECK: br label %{{.*}}, !llvm.loop ![[DISABLE:[0-9]+]]
 // CHECK: br i1 %{{.*}}, label %{{.*}}, label %{{.*}}, !llvm.loop ![[ENABLE:[0-9]+]]
+// CHECK: br label %{{.*}}, !llvm.loop ![[COUNT_TEMPLATE:[0-9]+]]
+// CHECK: br label %{{.*}}, !llvm.loop ![[DISABLE_TEMPLATE:[0-9]+]]
+
+template <int A>
+void unroll() {
+  [[clang::loop_unroll(A)]]
```

Comment:
Maybe it would be nice to add loop_unroll test for OpenCL code as well to validate that we don't break its processing.

Reply:
If I understand correctly what you mean, we already have tests for OpenCL unroll hint (can be found in `SemaOpenCL/` and `CodeGenOpenCL/`), which didn't fail in CI.

Reply:
Ah, sorry, I didn't find tests for opencl unroll attribute because searched for presence of 'loop_unroll' in opencl tests. However, this attribute has name 'opencl_unroll_hint'. Dropping request then.